modin/_compat/pandas_api/latest/general.py

```diff
@@ -15,9 +15,21 @@
 
 import pandas
 
-from modin.utils import _inherit_docstrings
+from modin.utils import _inherit_docstrings, to_pandas
 from modin.logging import enable_logging
 from modin.pandas import DataFrame
+from modin.error_message import ErrorMessage
+
+
+@_inherit_docstrings(pandas.from_dummies, apilink="pandas.from_dummies")
+@enable_logging
+def from_dummies(data, sep=None, default_category=None):
```

Comment:
We need issue to create tests for that function.

Reply:
will add

Reply:
https://github.com/modin-project/modin/issues/5085

Reply:
Thank you!